--- Resources/app/administration/build/plugins.vite.ts.orig
+++ Resources/app/administration/build/plugins.vite.ts
@@ -221,7 +221,7 @@
                 // For plugins
                 server = await createServer({
                     ...getBaseConfig(extension),
-                    server: { port },
+                    server: { port, host: true },
                 });

                 extensionInfoDebug(colors.green(`# Plugin "${extension.name}": Injected successfully`));
