--- Resources/app/administration/src/core/helper/retry.helper.spec.vue3.js.orig
+++ Resources/app/administration/src/core/helper/retry.helper.spec.vue3.js
@@ -0,0 +1,20 @@
+/**
+ * @package admin
+ */
+import retryHelper from './retry.helper';
+
+describe('core/helper/retry.helper.js', () => {
+    it('retries three times', async () => {
+        const innerFunction = jest.fn(() => {
+            throw new Error();
+        });
+
+        try {
+            await retryHelper.retry(innerFunction, 1, 1);
+        } catch (e) {
+            // nth
+        }
+
+        expect(innerFunction).toHaveBeenCalledTimes(2);
+    });
+});
