--- Theme/Command/ThemeDumpCommand.php.orig
+++ Theme/Command/ThemeDumpCommand.php
@@ -67,11 +67,10 @@
         $themeId = $input->getArgument('theme-id');
 
         if (!$themeId) {
-            $helper = $this->getHelper('question');
-
             $choices = $this->getThemeChoices();
 
-            if (\count($choices) > 1) {
+            if ($input->isInteractive() && \count($choices) > 1) {
+                $helper = $this->getHelper('question');
                 $question = new ChoiceQuestion('Please select a theme:', $choices);
                 $themeName = $helper->ask($input, $output, $question);
 
@@ -117,15 +116,19 @@
         $fs = $this->themeFilesystemResolver->getFilesystemForStorefrontConfig($themeConfig);
 
         $domainUrl = $input->getArgument('domain-url');
-        $domainUrl = $domainUrl ?? $this->askForDomainUrlIfMoreThanOneExists($themeEntity, $input, $output);
-
-        if ($domainUrl === null) {
-            $this->io->error(\sprintf('No domain URL for theme %s found', $themeEntity->getTechnicalName()));
-
-            return self::FAILURE;
-        }
-
-        $dump['domainUrl'] = $domainUrl;
+        if ($input->isInteractive()) {
+            $domainUrl = $domainUrl ?? $this->askForDomainUrlIfMoreThanOneExists($themeEntity, $input, $output);
+
+            if ($domainUrl === null) {
+                $this->io->error(\sprintf('No domain URL for theme %s found', $themeEntity->getTechnicalName()));
+
+                return self::FAILURE;
+            }
+        }
+
+        $dump['themeId'] = $themeEntity->getId();
+        $dump['technicalName'] = $themeConfig->getTechnicalName();
+        $dump['domainUrl'] = $domainUrl ?? '';
         $dump['basePath'] = $this->stripProjectDir($fs->location);
 
         file_put_contents(
@@ -144,7 +147,7 @@
     protected function getThemeChoices(): array
     {
         $choices = [];
-
+        /** @var ThemeCollection $themes */
         $themes = $this->themeRepository->search(new Criteria(), Context::createCLIContext())->getEntities();
 
         foreach ($themes as $theme) {
